### PR TITLE
ISSUE #1859 Top menu changes

### LINKED ISSUE #1859: Top menu changes
### Description
Reorganizing items in top menu

### Goals
- [x]  back to teamspace icon should just be the teamspace icon, not the exit looking icon
- [x]  user icon replaces the hamburger button, merging the 2 menus together  
- [x]  replace user icon with user's avatar (or initials if avatar doesn't exist)
- [x]  the menu should below the icon not replace, have the icon change when it's pressed (like gmail):

### PR DESCRIPTION
**Pull request by m1chalkubiak**
This fixes #1859

#### Description
- back to teamspace icon should just be the teamspace icon, not the exit looking icon
- user icon replaces the hamburger button, merging the 2 menus together
- replace user icon with user's avatar (or initials if avatar doesn't exist)
- the menu should below the icon not replace, have the icon change when it's pressed (like gmail):

### CODE REVIEW

**Comment by carmenfan** [comment]
@m1chalkubiak 
Looks good. Is it possible to remove the white outline on the picture? 
image

File: frontend/routes/app/app.component.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class App extends React.PureComponent<IProps, IState> {
 	});
 
 	public renderHeader = renderWhenTrue(() => (
-		<TopMenu onLogout={this.props.logout} onLogoClick={this.handleLogoClick} />
```
Comment (carmenfan) [inline frontend/routes/app/app.component.tsx]: Any reason why we're removing the logo click?

At the moment if you click on the logo it'll go to teamspace view. with this removed it no longer happens unless we're in the viewer.

Comment (m1chalkubiak) [reply frontend/routes/app/app.component.tsx]: Both `logout` action and `handleLogoClick` were only moved from `app.component` to `topMenu.component`, it doesn't have any impact on functionality it's just code optimization.

**Comment by m1chalkubiak** [comment]
> Unnamed avatar doesnt' seem to be showing:
> image

@carmenfan Can You give me access to this account? Lack of an avatar image should be handled properly, but maybe I missed something.

**Comment by carmenfan** [comment]
> > Unnamed avatar doesnt' seem to be showing:
> > image
> 
> @carmenfan Can You give me access to this account? Lack of an avatar image should be handled properly, but maybe I missed something.

The user is someCrazilyLongNameLikeThis

the password is changed to 123 (You could just do this in the future, the database we gave you is solely for your testing, we don't use it at all)

**Comment by m1chalkubiak** [comment]
> 
> The user is someCrazilyLongNameLikeThis
> 
> the password is changed to 123 (You could just do this in the future, the database we gave you is solely for your testing, we don't use it at all)

Thanks, sorry I constantly forget about it. I found the issue, we were always trying to get the avatar image from the API, even when there was the API response with the `hasAvatar: false` property. Fixed.

**Pull request merged**